card/lib/card/format/nest/fetch.rb

```diff
@@ -7,6 +7,7 @@ def fetch_nested_card cardish, opts={}
           case cardish
           when Card            then cardish
           when Symbol, Integer then Card.fetch cardish
+          when "_", "_self"    then card.context_card
```

Comment:
Is there a good reason that we used to refetch instead of using the same card? That caused problems for rendering trashed cards (which happens for notification emails of deletions) because "{{_self}}" refers then to a new card instead of the trashed card.